src/main/java/com/ft/methodearticleinternalcomponentsmapper/exception/UntransformableMethodeContentException.java

```diff
@@ -0,0 +1,22 @@
+package com.ft.methodearticleinternalcomponentsmapper.exception;
+
+/**
+ * The post is a valid WordPress post, but cannot be published by the transformer.
+ * For example:
+ * <ul>
+ * <li>it has a type or custom type other than "post", meaning it has custom formatting associated with it</li>
+ * <li>it has no body text</li>
+ * </ul>
+ */
+public class UntransformableMethodeContentException extends RuntimeException {
```

Comment:
Since "unstransformable" is not a word maybe we could change this exception to InvalidMethodeContentException?